DomainContent/Zombies/ZombieManager/zombiePositions.js

```diff
@@ -0,0 +1,292 @@
+	            y:-0.025,
+	            z:-61.34
+	        },
+	        4 : {
+	            x : -1.9797,
+	            y : -10.2699,
+	            z : 22.4347
+	        },
+	        readable : {
+	            1 : "Back Alley",
+	            2 : "Across from Gas Main",
+	            3 : "Forest",
+	            4 : "Boardwalk"
+	        }
+	    },
+	    Maroon: {
+	        1 : {
+	            x:-21.4689,
+	            y:-9.1611,
+	            z:4.42181
+	        },
+	        2 : {
+	            x:-19.5104,
+	            y:-9.16155,
+	            z:-8.32982
+	        },
+	        3 : {
+	            x:34.951,
+	            y:-2.9243,
+	            z:-45.204
+	        },
+	        readable : {
+	            1 : "Bus",
+	            2 : "Back Alley",
+	            3 : "Plateau"
+	        }
+	    },
+	    Boardwalk : {
+	        1 : {
+	            x:-36.7870,
+	            y:-9.01325,
+	            z:18.25436
+	        }, 
+	        2 : {
+	            x:-28.024,
+	            y:-9.3441,
+	            z:28.9311
+	        },
+	        3 : {
+	            x : -1.9797,
+	            y : -10.2699,
+	            z : 22.4347
+	        },
+	        readable : {
+	            1 : "Blocking cars",
+	            2 : "Boardwalk path",
+	            3 : "Final boardwalk"
+	        }
+	    },
+	    Generator : {
+	        1 : {
+	            x:3.25877,
+	            y:-7.6755,
+	            z:-22.40
+	        },
+	        2 : {
+	            x:2.4047,
+	            y:-6.2222,
+	            z:-27.292
+	        },
+	        3 : {
+	            x:3.25877,
+	            y:-7.6755,
+	            z:-22.40
+	        },
+	        4 : {
+	            x:1.0234,
+	            y:-9.2155,
+	            z:21.691
+	        },
+	        readable : {
+	            1 : "Hill by A&J",
+	            2 : "Near statue at A&J",
+	            3 : "Hill by A&J",
+	            4 : "Boardwalk"
+	        }
+	    },
+	    Plateau : {
+	        1 : {
+	            x:26.7789,
+	            y:-3.0295,
+	            z:-23.85
+	        },
+	        2 : {
+	            x:35.30,
+	            y:-2.96,
+	            z:-45.07
+	        },
+	        readable : {
+	            1 : "Plateau Entry",
+	            2 : "Top of Hill"
+	        }
+	    },
+	    CityWalk : {
+	        1 : {
+	            x:-23.48978,
+	            y:-3.431381,
+	            z:-41.3309
+	        },
+	        2 : {
+	            x:-10.348,
+	            y:-4.2053,
+	            z:-33.520
+	        },
+	        3 : {
+	            x:1.0234,
+	            y:-9.2155,
+	            z:21.691
+	        },
+	        readable : {
+	            1 : "Before Regalia",
+	            2 : "Near Regalia Entrance",
+	            3 : "Boardwalk"
+	        }
+		}
+	},
+
+	getPosition: function(color, number, library) {
+		print("Retrieving position " + color + " " + number);
+		if (color in library) {
+			var colorJSON = library[color];
+			if (number in colorJSON) {
+				return colorJSON[number];
+			}
+			print("number not in specified color");
```

Comment:
You can keep these error print statements. Since they're not spammy and actually help identify problems.